src/frontend/components/PlaylistPortability/index.tsx

```diff
@@ -0,0 +1,299 @@
+    id: 'component.PlaylistPortability.loadingPlaylist',
+  },
+  playlistTitle: {
+    defaultMessage: 'Belongs to playlist {title} ({id})',
+    description: 'Title for the current playlist.',
+    id: 'component.PlaylistPortability.playlistTitle',
+  },
+  sharedListTitle: {
+    defaultMessage: 'Playlist shared with:',
+    description: '',
+    id: 'component.PlaylistPortability.sharedListTitle',
+  },
+  shareWithPlaylist: {
+    defaultMessage: 'Share with another playlist',
+    description: 'Input label for adding a new portability.',
+    id: 'component.PlaylistPortability.shareWithPlaylist',
+  },
+  shareWithPlaylistDetails: {
+    defaultMessage:
+      'Sharing this playlist will grant access to all this resources.',
+    description: 'Helper text to explain what sharing will do.',
+    id: 'component.PlaylistPortability.shareWithPlaylistDetails',
+  },
+  shareWithPlaylistPlaceholder: {
+    defaultMessage: 'Paste playlist uuid',
+    description: 'Helper text for adding a new portability.',
+    id: 'component.PlaylistPortability.shareWithPlaylistPlaceholder',
+  },
+  updatePlaylistSuccess: {
+    defaultMessage: 'Playlist updated.',
+    description: 'Message when playlist update is successful.',
+    id: 'component.PlaylistPortability.updatePlaylistSuccess',
+  },
+  updatePlaylistFail: {
+    defaultMessage: 'Playlist not updated!',
+    description: 'Message when playlist update failed.',
+    id: 'component.PlaylistPortability.updatePlaylistFail',
+  },
+  addPortability: {
+    defaultMessage: 'add share',
+    description: 'Accessible message for adding a portability.',
+    id: 'component.PlaylistPortability.addPortability',
+  },
+  removePortability: {
+    defaultMessage: 'Delete sharing with {title}',
+    description: 'Accessible message for removing a portability.',
+    id: 'component.PlaylistPortability.removePortability',
+  },
+  hasPortabilityWith: {
+    defaultMessage: 'Shared with {title}',
+    description:
+      'Accessible message for displaying a portability with a playlist.',
+    id: 'component.PlaylistPortability.hasPortabilityWith',
+  },
+  copied: {
+    defaultMessage: '{text} copied!',
+    description: 'Message displayed when playlist info are copied.',
+    id: 'components.PlaylistPortability.copied',
+  },
+  copy: {
+    defaultMessage: 'Copy key {text}.',
+    description: 'Accessible message for copying a playlist id.',
+    id: 'components.PlaylistPortability.copy',
+  },
+});
+
+interface PlaylistPortabilityListProps {
+  playlist: Playlist;
+  removePlaylistPortability: (oldPortabilityID: string) => void;
+}
+
+export const PlaylistPortabilityList = ({
+  playlist,
+  removePlaylistPortability,
+}: PlaylistPortabilityListProps) => {
+  const intl = useIntl();
+  if (playlist!.portable_to!.length > 0) {
+    return (
+      <Box>
+        <Text>
+          <FormattedMessage {...messages.sharedListTitle} />
+        </Text>
+        <List
+          data={playlist?.portable_to}
```

Comment:
According to your props, playlist type is neither undefined or null. So you should write:
```suggestion
          data={playlist.portable_to}
```